treeherder/etl/management/commands/pulse_listener.py

```diff
@@ -17,6 +17,9 @@ class Command(BaseCommand):
     help = "Read tasks and pushes from a set of pulse exchanges and queue for ingestion"
 
     def handle(self, *args, **options):
+        if(SKIP_INGESTION):
+            self.stdout.write("Skipping ingestion of Pulse Tasks")
```

Comment:
Could you please use a logger?
You can see an example [here](https://github.com/mozilla/treeherder/blob/c702d2b54805ac7175b46e6618b4f7b670a6dcc8/treeherder/autoclassify/tasks.py)

Make it an info message. Thanks!